commands/version/index.js

```diff
@@ -556,6 +578,32 @@ class VersionCommand extends Command {
 
     return gitPush(this.gitRemote, this.currentBranch, this.execOpts);
   }
+
+  createGitHubReleases() {
+    this.logger.info("github", "Creating releases...");
+
+    const client = createGitHubClient();
+
+    return Promise.all(
+      this.releaseNotes.map(({ notes, name }) => {
+        const tag = name === "root" ? this.tags[0] : this.tags.find(t => t.startsWith(name));
```

Comment:
I don't think you need to create releases for every package when the versioning is _not_ independent (i.e., "fixed"). You would only need to create one release, the "root", in that case.

Reply:
When "fixed", this should only have 1 value in `releaseNotes` correct? Since the chain is only happening once, and not on every package.